sim/xcelium/__init__.py

```diff
@@ -0,0 +1,394 @@
+#!/usr/bin/env python3
+
+# HAMMER-VLSI PLUGIN, XCELIUM 
+# Notes: This plugin sets up xrun to execute in a three-step xrun invocation.
+#        This bridges multi-tool direct-invocation and the xrun single-invocation.
+
+
+import os
+import re
+import shutil
+import json
+import datetime
+import io
+import sys
+from typing import Dict, List, Optional, Callable, Tuple
+from multiprocessing import Process
+
+import hammer_utils
+import hammer_tech
+from hammer_tech import HammerTechnologyUtils
+from hammer_vlsi import FlowLevel, TimeValue
+from hammer_vlsi import HammerSimTool, HammerToolStep, HammerLSFSubmitCommand, HammerLSFSettings
+from hammer_logging import HammerVLSILogging
+
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)),"../../common"))
+from tool import CadenceTool
+
+class xcelium(HammerSimTool, CadenceTool):
+
+  @property
+  def xcelium_ext(self) -> List[str]:
```

Comment:
Similar to comment about waveforms below, this should eventually be absorbed into the graph methodology @bdngo